src/main/java/org/jabref/gui/EntryTypeViewModel.java

```diff
@@ -52,6 +58,9 @@ public EntryTypeViewModel(JabRefPreferences preferences, BasePanel basePanel, Di
         fetchers.addAll(WebFetchers.getIdBasedFetchers(preferences.getImportFormatPreferences()));
         selectedItemProperty.setValue(getLastSelectedFetcher());
 
+        Predicate<String> notEmpty = input -> (input != null) && !input.trim().isEmpty();
```

Comment:
This can be replaced by `StringUtil#isNotBlank`

Reply:
Done